ui/imports/utils/Global.qml

```diff
@@ -14,6 +14,7 @@ Item {
     property bool popupOpened: false
     property int settingsSubsection: Constants.settingsSubsection.profile
 
+    property var globalUtilsInst: globalUtils
```

Comment:
I noticed that warning `file:///home/michalc/workspace/status-desktop/ui/imports/utils/Global.qml:17: ReferenceError: globalUtils is not defined` can be eliminated by checking `globalUtils` this way:
```
property var globalUtilsInst: typeof(globalUtils) === "undefined" ? null : globalUtils
```
alternatively it could be left unassigned in `Global.qml` and assigning it in `main.qml` as its done for `mainModuleInst` (`main.qml`, line 144: `Global.mainModuleInst = mainModule`)

Reply:
Ideally this would be:
```qml
property var globalUtilsInst: globalUtils ?? null
```
but that's for Qt 5.15 :/